vuepress/docs/next/tutorials/consume/multitenancy.md

```diff
@@ -165,24 +164,36 @@ kubectl scale deploy/YOUR-APP-NAME-deployment --replicas=1 -n YOUR-NAMESPACE
 ```
 4. Confirm that the secondary tenant is working correctly. This may include testing the EntandoApp itself (including digital assets delivered via CDS), the AppBuilder, and enterprise search for Solr. The tutorials for each service include verification steps that can be followed once the tenant configuration is fully in place.
 
+## Bundles
+Once tenants are in order, [Entando Bundles](../../docs/curate/bundle-details.md) can be deployed to each tenant independently, whether created anew or deployed from registries added in the App Builder. 
+
+When bundles are installed to any tenant, the Component Manager inserts an `ENTANDO_TENANT_CODE`, an environment variable related to the tenant domain name, identifying which tenant it belongs to. 
```

Comment:
Could it be more precise `the Component Manager injects an `ENTANDO_TENANT_CODE` into every plugin microservice`?